prdoc/schema_user.json

```diff
@@ -0,0 +1,223 @@
+
+          {"const": "Runtime Dev",
+          "title": "All of those who rely on the runtime. A parachain team that is using a pallet. A DApp that is using a pallet. These are people who care about the protocol (WASM), not the meta-protocol (client)."},
+
+          {"const": "Node Operator",
+          "title": "Those who don't write any code and only run code."},
+
+          {"const": "Runtime User",
+          "title": "Token holders who don't run anything but are involved in the network."}
+        ]
+      },
+      "crate": {
+        "type": "object",
+        "additionalProperties": false,
+        "properties": {
+          "name": {
+            "type": "string"
+          },
+          "semver": {
+            "$ref": "#/$defs/semver"
+          },
+          "note": {
+            "type": "string"
+          }
+        }
+      },
+      "semver": {
+        "type": "string",
+        "enum": [
+          "major",
+          "minor",
+          "patch",
+          "no change"
+        ]
+      },
+      "migration_db": {
+        "type": "object",
+        "properties": {
+          "name": {
+            "type": "string"
+          },
+          "description": {
+            "type": "string"
+          }
+        },
+        "additionalProperties": false,
+        "required": [
+          "name",
+          "description"
+        ]
+      },
+      "migration_runtime": {
+        "type": "object",
+        "properties": {
+          "pallet": {
+            "type": "string"
+          },
+          "description": {
+            "type": "string"
+          }
+        },
+        "additionalProperties": false,
+        "required": [
+          "pallet",
+          "description"
+        ]
+      },
+      "doc": {
+        "type": "object",
+        "description": "You have the the option to provide different description of your PR for different audiences.",
+        "additionalProperties": false,
+        "properties": {
+          "audience": {
+            "description": "The selected audience",
+            "$ref": "#/$defs/audience"
+          },
+          "title": {
+            "type": "string",
+            "description": "Optional title override for the PR and for the current audience"
+          },
+          "description": {
+            "description": "Description of the change",
+            "type": "string"
+          },
+          "notes": {
+            "description": "Optional relevant note",
+            "$ref": "#/$defs/array_of_strings"
+          }
+        }
+      },
+      "array_of_strings": {
+        "description": "An array of strings that can be empty",
+        "type": "array",
+        "items": {
+          "type": "string"
+        }
+      },
+      "host_function": {
+        "type": "object",
+        "additionalProperties": false,
+        "description": "List of host functions and their descriptions",
+        "properties": {
+          "name": {
+            "type": "string"
```

Comment:
What would be the name? 

I'm still not convinced that this should be more than a `bool`.

Reply:
A sensible choice would be the name of the host_function to allow readers having a starting point to seach the code/rustdoc. For now, the content would look like:
```
host_functions:
   - name: RuntimeTasks::spawn
     description: Wasm host function for spawning task.
     notes:
        - This should not be used directly. Use `sp_tasks::spawn` instead.
```

Reply:
Host functions will not be added anymore without a RFC. Thus, we should not need to explain it here. More just mention that it was added.

Reply:
Sure, the addition of that field is based on the past but if this barely ever happen again, I can drop the entire property and it will have to be mentioned in the audience.